docs/reference/adobe/experience/customerJourneyManagement/messageexecution.schema.json

```diff
@@ -52,6 +52,11 @@
                     "title": "Journey Action ID",
                     "type": "string",
                     "description": "Journey Action ID, for which MessageExecution is triggered."
+                },
+                "https://ns.adobe.com/experience/customerJourneyManagement/messageExecution/emailNamespace": {
```

Comment:
Why we require this field to be a part of Feedback/ExpEvent event. Can't consent service makes call to Message runtime to find the emailNamespace for corresponding MessageExecution?

Reply:
Message Runtime already populates this field in "pid" payload in most URLs including unsubscribe links, which applies to the link use case for list-unsubscribe. Let's keep it consistent and provide it in the mailto use case as well.

Reply:
@sawhneyh  Consent service does not directly interface with message runtime. All the message context is passed to them from external sources. MR passes the required metadata to ESS-EFS which then flows to downstream systems.
@guayang-adobe  Even if pid is passed in mailTo, EFS still has to process it & set the fields in xdm as communication between consent & efs is via xdm event. So, we would still need to introduce a field for email namespace.

Reply:
The reason I am not comfortable using XDM field for communicating the emailNamespace to the consent service, is that this data will also be ingested in AEP. The end user can also view this data, and it may be confusing why this field exist in XDM.